site/content/en/spec-3.3.md

```diff
@@ -230,6 +230,9 @@ A `REUSE.toml` file MAY be located in any directory, and can cover files that ar
 within its directory or deeper. You MAY have multiple `REUSE.toml` files in
 different directories.
 
+If the Project uses a VCS, the `REUSE.toml` files MUST NOT be ignored by the
+VCS. Any `REUSE.toml` file ignored by VCS is ignored by REUSE.
```

Comment:
Using “ignored by REUSE” is not ideal wording here.

https://github.com/fsfe/reuse-website/blob/3c51fc6ab1e017e881e7dc7a42bace40a80fc007/site/content/en/spec-3.3.md?plain=1#L79 explicitly says `REUSE.toml` files are ignored already.

Reply:
Rephrased